CMakeLists.txt

```diff
@@ -73,15 +73,12 @@ endif()
 #------------------------------------------------------------------------------#
 option(Legion_USE_GASNet "Enable the distributed GASNet backend" OFF)
 if(Legion_USE_GASNet)
-  set(GASNet_THREADING "par" CACHE STRING "GASNet Threading model to use")
-  set_property(CACHE GASNet_THREADING PROPERTY STRINGS seq par parsync)
-  mark_as_advanced(GASNet_THREADING)
+  set(GASNet_THREADING par)
+  set(GASNet_PREFERRED_CONDUITS gemeni aries ibv)
```

Comment:
This looks like a speeling error on "gemini"?  Also, let's list aries before gemini - I'm not sure if Aries-based systems claim to support gemini or not.